spec/vsphere_connection_handling_spec.rb

```diff
@@ -35,9 +35,9 @@
   end
 
   describe '#conn_opts' do
-    let(:ui) { double( 'Ui', ask: 'passwurd') }
+    let(:ui) { double('Ui', ask: 'passwurd') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.